### PR TITLE
gruvbox-kvantum: init at 1.1

### PR DESCRIPTION
**Pull request by istudyatuni**
## Things done

Gruvbox theme for Kvantum: https://github.com/sachnr/gruvbox-kvantum-themes

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md). (I hope)

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/gr/gruvbox-kvantum/package.nix [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      runHook postInstall
+    '';
+
+    meta = with lib; {
+      description = "Gruvbox themes for kvantum";
+      homepage = "https://github.com/sachnr/gruvbox-kvantum-themes";
+      license = licenses.gpl3;
+      platforms = platforms.linux;
+      maintainers = [ ];
+    };
```
Comment (FliegendeWurst) [inline pkgs/by-name/gr/gruvbox-kvantum/package.nix:46]:
```suggestion
    meta = {
      description = "Gruvbox themes for kvantum";
      homepage = "https://github.com/sachnr/gruvbox-kvantum-themes";
      license = lib.licenses.gpl3;
      platforms = lib.platforms.linux;
      maintainers = [ ];
    };
```

**Review by FliegendeWurst — CHANGES_REQUESTED** [review]

File: pkgs/by-name/gr/gruvbox-kvantum/package.nix [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      cp -a "${variant}" $out/share/Kvantum
+      runHook postInstall
+    '';
+
+    meta = with lib; {
+      description = "Gruvbox themes for kvantum";
+      homepage = "https://github.com/sachnr/gruvbox-kvantum-themes";
+      license = licenses.gpl3;
+      platforms = platforms.linux;
+      maintainers = [ ];
```
Comment (Aleksanaa) [inline pkgs/by-name/gr/gruvbox-kvantum/package.nix:45]: new packages must have maintainers. Add yourself to `maintainers/maintainer-list.nix` if you aren't one.

Comment (istudyatuni) [reply pkgs/by-name/gr/gruvbox-kvantum/package.nix:45]: I haven't seen this requirement

Comment (istudyatuni) [reply pkgs/by-name/gr/gruvbox-kvantum/package.nix:45]: I only saw

> We encourage people who care about a package to assign themselves as a maintainer

and I'm not sure, if I want to be the one, so I didn't add myself here

Comment (Aleksanaa) [reply pkgs/by-name/gr/gruvbox-kvantum/package.nix:45]: https://github.com/NixOS/nixpkgs/tree/master/pkgs#quick-start-to-adding-a-package

https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes

File: maintainers/maintainer-list.nix
[the remark this answers is not in the document]
Comment (istudyatuni) [reply maintainers/maintainer-list.nix:1]: I hope this won't cause massive ping (probably I misunderstand something)

**Comment by istudyatuni** [comment]
I think I will add 2 separate parameters for configuring, I don't like current variants. But a bit later

**Comment by istudyatuni** [comment]
Okay, I won't change variants, current variant is simpler to configure within home-manager
```nix
let
  variant = "Gruvbox-Dark-Brown";
  pkg = pkgs.gruvbox-kvantum.override {inherit variant;};

  dir = ".config/Kvantum";
in {
  home.file."${dir}/kvantum.kvconfig".source = util.toINI "kvantum.kvconfig" {
    General.theme = variant;
  };
  home.file."${dir}/${variant}".source = "${pkg}/share/Kvantum/${variant}";
}
```

**Review by Aleksanaa — APPROVED** [review]

**Pull request merged**